README.md

```diff
@@ -833,3 +834,25 @@ You will have to enter the sudo password for your computer twice (once for the b
   ![Tutorial 1 icon](docs/images/tutorial_1_icon.png)
  
   ![Tutorial 1 result](docs/images/tutorial_1_result.png)
+
+# Troubleshooting #
+
+1. SSH warnings when using the same laptops for multiple different NUCs (in server_and_nuc_deploy and teleop_deploy): for a given server laptop, only 1 NUC is supposed to be used. If the NUC is changed, the SSH keys stored on the laptop don't match the NUC, so aurora has to be re-run. In this case, it's required that the user manually deletes the .ssh folder on the server laptop to clear ssh keys.
```

Comment:
only one aesthetic comment. Could you make bold or underline the point of the troubleshooting

e.g **ssh warning:**     it makes it way more readable in my opinion

Reply:
Done in the latest commit